run-tests.php

```diff
@@ -3014,7 +2985,7 @@ function compute_summary()
     }
 }
 
-function get_summary($show_ext_summary, $show_html)
+function get_summary($show_ext_summary)
```

Comment:
There are some get_summary() calls that still pass the second parameter.

Reply:
resolved.